7_react/react2/week1/my-app/src/Who.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function WhoAreWe(){
+    return(
+    <div>
+    <h2 className='header'>
+    Who we are
+    </h2>
+<p>
+HackYourFuture is a not-for-profit coding school for people with limited access to education and the labor market. The majority of our students worldwide are refugees.

```

Comment:
Remember to always format your code before committing to the repository. You can set VS Code to do this automatically every time you save, for example.